package.json

```diff
@@ -18,12 +18,13 @@
     "format": "sf-format",
     "lint": "sf-lint",
     "lint-fix": "yarn sf-lint --fix",
-    "prepack": "sf-build",
     "postpack": "shx rm -f oclif.manifest.json",
+    "prepack": "sf-prepack",
+    "prepare": "sf-install",
     "pretest": "sf-compile-test",
-    "test": "sf-test --require ts-node/register",
```

Comment:
`ts-node/register` is already specified in [.mocharc.json](https://github.com/forcedotcom/source-tracking/blob/1ff40a580bbd9f70364756e67bc75f3cd767196b/.mocharc.json#L2)